unity/ComputerVision-UnityClient/Assets/Scripts/AppManager.cs

```diff
@@ -0,0 +1,370 @@
+using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+using UnityEngine.Networking;
+using System.Threading.Tasks;
+using System;
+
+namespace MobiledgeXComputerVision
+{
+    public class AppManager : MonoBehaviour
+    {
+        public RawImage rawImage;
+        public Texture rectTexture;
+        public Texture nodeTexture;
+        public static bool showGUI;
+        int[][] faceDetectionRects;
+        int[] faceRecognitionRect;
+        static string faceRecognitionSubject;
+        static float faceRecognitionConfidenceLevel;
+        static @Object[] objectsDetected;
+        float imgScalingFactor;
+        public static int level = 0;
+        public NetworkManager networkManager;
+        bool serviceAlreadyStarted
+        {
+            get
+            {
+                switch (serviceMode)
+                {
+
+                    case ServiceMode.FaceRecognition:
+                        return faceRecognitionRect == null ? false : true;
+                    case ServiceMode.ObjectDetection:
+                        return objectsDetected == null ? false : true;
+                    default:
+                    case ServiceMode.FaceDetection:
+                        return faceDetectionRects == null ? false : true;
+                }
+
+            }
+        }
+
+        public static string urlSuffix
+        {
+            get
+            {
+                switch (serviceMode)
+                {
+                    case ServiceMode.FaceRecognition:
+                        return "/recognizer/predict/";
+                    case ServiceMode.ObjectDetection:
+                        return "/object/detect/";
+                    default:
+                    case ServiceMode.FaceDetection:
+                        return "/detector/detect/";
```

Comment:
If serviceMode doesn't match any of these, it will use "/detector/detect/". Should there be an error instead?

Reply:
So default here is necessary, because it needs to return a value, I guess I will return an empty string, which will throw an error when the request is made.

Reply:
fixed